translate/rus.json

```diff
@@ -370,7 +371,7 @@
 "tabs per row":"вкладки в строке",
 "target color":"цвет цели",
 "TB Hits":"Позиции ЭТ",
-"TB":"TB",
+"TB":"ЭТ",
```

Comment:
We have this acronym translated in 'TB hits', why shouldn't it be done the same here?:)